tests/gocase/unit/slowlog/slowlog_test.go

```diff
@@ -39,13 +39,13 @@ func TestSlowlog(t *testing.T) {
 	defer func() { require.NoError(t, rdb.Close()) }()
 
 	t.Run("SLOWLOG - check that it starts with an empty log", func(t *testing.T) {
-		require.EqualValues(t, 0, rdb.Do(ctx, "slowlog", "len").Val())
+		require.EqualValues(t, 0, len(rdb.SlowLogGet(ctx, -1).Val()))
 	})
 
 	t.Run("SLOWLOG - only logs commands taking more time than specified", func(t *testing.T) {
 		require.NoError(t, rdb.ConfigSet(ctx, "slowlog-log-slower-than", "100000").Err())
 		require.NoError(t, rdb.Ping(ctx).Err())
-		require.EqualValues(t, 0, rdb.Do(ctx, "slowlog", "len").Val())
```

Comment:
Although I think using `SLOWLOG LEN` fits the original code and covers most functionality, these tests can be OK to merge from my perspective.

Merging...